scripts/linter.sh

```
@@ -2,7 +2,7 @@
 
 MINLINTSCORE=10
 
-if ! (pylint --fail-under=$MINLINTSCORE --extension-pkg-whitelist=cv2 peekingduck); then
+if ! (pylint --fail-under=$MINLINTSCORE --extension-pkg-whitelist=cv2 --disable=W0511 peekingduck); then
```

Although this is added to suppress error due to `FIXME` comments during development, it should be removed after development is done

---

Noted, removed.